flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/LoadConstants.java

```diff
@@ -33,4 +33,6 @@ public class LoadConstants {
     public static final String READ_JSON_BY_LINE = "read_json_by_line";
     public static final String GROUP_COMMIT = "group_commit";
     public static final String GROUP_COMMIT_OFF_MODE = "off_mode";
+    public static final String COMPRESS_TYPE = "compress_type";
+    public static final String COMPRESS_TYPE_GZ = "gz";
```

Comment:
Is it more clear to use `gzip` here?

Reply:
This depends on streamload and must be gz.
![image](https://github.com/user-attachments/assets/e9656112-9dd7-408b-908c-603c0d3698e4)